### DISCUSSION TITLE
Carbon Custom Theme mixin

### DISCUSSION THREAD

**Discussion by tsu84** [carbon-design-system/carbon#11663] [Help]
Hi,

I'm trying to customise the carbon theme following the [documentation](https://carbondesignsystem.com/guidelines/themes/overview#customizing-a-theme), but I can't figure out how to set up the mixin to make it work.

Can someone explain me with an example?

**Comment by tay1orjones** [comment]
Hey @tsu84, the theme docs are a bit out of date, https://github.com/carbon-design-system/carbon/issues/11293

We're in the midst of updating that documentation to explain usage with `@carbon/react` and `@carbon/styles` - it hasn't been merged quite yet, but I think it will help you:

Website: https://github.com/carbon-design-system/carbon-website/pull/2998
Package readme/docs: https://github.com/carbon-design-system/carbon/pull/11635

**Reply by tsu84** [reply]
@tay1orjones thanks for the reply.

I've read your references, but still I can't figure out how to override with a custom theme mixin, or change the value of a token and make it work.

When the sass compilation exits without errors, custom rules are not applied in css file.

**Reply by tay1orjones** [reply]
@tsu84 Sorry I missed your reply! I'm happy to take a closer look if you could share a repro of what you're trying to accomplish. We've got templates in [codesandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/v11.6.0/examples/codesandbox-with-sass-compilation/) and [stackblitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/examples/vite) (usually a bit faster compilation than codesandbox)
